frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -79,3 +84,4 @@ frontend/packages/dev-console/integration-tests/
 1. Don't use static sleep statements (browser.sleep)
 2. Comments should be included wherever required
 3. Don't inlcude console.log statements while raising PR
+4. .gherkin-lintrc  configuration file present in forntend folder is used to set Gherkin standards
```

Comment:
Spelling: `configuration file present in frontend...`

Reply:
updated